editor.planx.uk/src/@planx/components/DrawBoundary/Public/index.tsx

```diff
@@ -218,6 +218,10 @@ export default function Component(props: Props) {
           boundary && props.dataFieldBoundary ? boundary : undefined,
         [props.dataFieldArea]:
           boundary && props.dataFieldBoundary ? area : undefined,
+        [`${props.dataFieldArea}.hectares`]:
+          boundary && area && props.dataFieldBoundary
+            ? area / 10000
```

Comment:
Google agrees so it must be true

![image](https://user-images.githubusercontent.com/7684574/204620731-70bbf3fd-1fa2-4738-95ab-7aaaf5084fdc.png)